app/styles/progress-bar.css

```diff
@@ -0,0 +1,74 @@
+:root {
+  --progress-bar-color: rgba(0, 128, 0, 0.9);
+  --light-grey:rgba(128, 128, 128, 0.5);
+  --black: black;
+  --white: white;
+}
+.progress-bar-container {
+  display: flex;
+  gap: 4px;
+}
+
+.progress-bar__button {
+  width: 1.4rem;
+  height: 1.1rem;
+  font-size: 60%;
```

Comment:
- please use definite values instead of percentages for font size, as percentage can have very different values across devices and screen resolutions.

Reply:
made required changes.